cluster/gce/gci/configure.sh

```diff
@@ -424,15 +418,15 @@ function install-docker {
     lsb-release
 
   # Add the Docker apt-repository
-  curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg \
+  curl -fsSL https://download.docker.com/linux/"$(. /etc/os-release; echo "$ID")"/gpg \
```

Comment:
I think it may be better to move the double quotes around the whole URL here:

```
curl -fsSL "https://download.docker.com/linux/$(. /etc/os-release; echo "${ID}")/gpg" \
```

Reply:
Sure thing. In some projects, people want it this way, only quoting what can actually contain whitespaces and such, while in other projects they want what you suggest. To me, anything goes. I'll update as per your suggestion!